actionpack/lib/action_dispatch.rb

```diff
@@ -94,6 +94,19 @@ module Session
     autoload :CookieStore,         "action_dispatch/middleware/session/cookie_store"
     autoload :MemCacheStore,       "action_dispatch/middleware/session/mem_cache_store"
     autoload :CacheStore,          "action_dispatch/middleware/session/cache_store"
+
+    def self.resolve_store(session_store)
+      self.const_get(session_store.to_s.camelize)
+    rescue NameError
+      raise <<~ERROR
+        Unable to resolve session store :#{session_store}.
+
+        :#{session_store} resolves to ActionDispatch::Session::#{session_store.to_s.camelize},
+        but that class is undefined.
+
+        Is :#{session_store} spelled correctly, and are the necessary gems installed?
+      ERROR
+    end
```

Comment:
I like this extraction! :heart_eyes: 

I wanted to point out a subtle difference between `":#{session_store}"` and `"#{session_store.inspect}"`.  Symbols can include arbitrary characters, including slashes which are validly "camelized" to `"::"`:

```ruby
symbol = :"my_namespace/my_store"

puts symbol.to_s.camelize
# => MyNamespace::MyStore

puts ":#{symbol} versus #{symbol.inspect}"
# => :my_namespace/my_store versus :"my_namespace/my_store"
```

So it might be better to use `session_store.inspect` here.